### ISSUE TITLE
Feature Request : Geofence Amount

### ISSUE THREAD

**Issue by jaimzj** [traccar/traccar#4278]
A feature to add amount values to a Geofence. 

A section in the add geofence dialog where user can add the following values.
with ability to make multiple entries based on device category.
- device category (car, truck, bus, bike, motorcycle, pet, person, tractor....)
- amount = 00.00
- expenseType = (toll / parking / entry fee)

Each time a device crosses (enters the geofence) the amount based on the device category is added to that event.

These can be helpful to keep tap on charges being pilled up on a particular device (expenseType) based on which businesses/users can take business or informative decisions.

**Comment by parthibd** [comment]
This feature has nothing to do with the core tracking features that traccar provides . You should architecture you application to use traccar as a microservice .

**Comment by jaimzj** [comment]
I disagree, not all features need to have something to do with the core tracking features of traccar, examples would be the driver feature or maintenance feature just to name few.

It all trickles down to the value add a feature can provide to a specific set or majority user base of traccar, who use the official traccar.

your suggestion applies for users who have their own application or use traccar API's etc,

**Comment by parthibd** [comment]
Tracccar is already open source. So anyone is free to implement the features they need by changing the source accordingly.

As you meant drivers, maintenance and in fact all the features that are there in traccar are built around the same core principles. I do agree to your view that those features will add more value to traccar.

You can make the proposal, but I don't believe those features will make their way into any coming releases soon if not at all. 

We are free to modify tracccar, and that's exactly what most companies are doing.

**Comment by jaimzj** [comment]
I am aware of open source, free to modify, etc etc. !

**Comment by parthibd** [comment]
Why don't you make a pull request and see where it goes? ☺️
